presto-product-tests/conf/presto/etc/multinode-master.properties

```diff
@@ -14,3 +14,7 @@ http-server.http.port=8080
 query.max-memory=1GB
 discovery-server.enabled=true
 discovery.uri=http://presto-master:8080
+experimental.spill-enabled=true
```

Comment:
I don't think it's a good idea to run tests with spill.
If anything, it would be better to run fewer/smaller tests instead.